.gitignore

```diff
@@ -1,4 +1,4 @@
 .DS_Store
 .idea
 documents
-*.toml
```

Comment:
What's the reason for the removal of the new line?

Reply:
I was editing the .gitignore but decided against the entries. This was purely an artifact of that modification and subsequent manual reversion.